Extension/src/LanguageServer/client.ts

```diff
@@ -574,6 +598,128 @@ export class DefaultClient implements Client {
                         }
                     }
                     this.disposables.push(vscode.languages.registerWorkspaceSymbolProvider(new WorkspaceSymbolProvider(this)));
+
+                    class FindAllReferencesProvider implements vscode.ReferenceProvider {
```

Comment:
Put FindAllReferencesProvider  in different file?

Reply:
Same as above.  The other 3 as well.  But, there are some complications I'd like to address after Insiders.